packages/message/src/Granules.ts

```diff
@@ -8,18 +8,35 @@
  * const Granules = require('@cumulus/message/Granules');
  */
 
-import get from 'lodash/get';
 import { Message } from '@cumulus/types';
 
+interface MessageWithGranules extends Message.CumulusMessage {
+  payload: {
+    granules?: object[]
+  }
+}
+
 /**
- * Get granules from execution message.
+ * Get granules from a workflow message.
  *
- * @param {Message.CumulusMessage} message - An execution message
+ * @param {MessageWithGranules} message - A workflow message
```

Comment:
Are workflow and execution interchangeable?

Reply:
Sort of. "Workflow" is the term we use generically in Cumulus for a Step Function composed of tasks. And an execution is an instance of running that workflow. I suppose that an "execution" message is a more accurate description

Reply:
I see. Even though `execution` message is technically more accurate, I think I do prefer `workflow` as you have it because it ties back to the cumulus vocabulary and provides more context.